metecho/api/serializers.py

```diff
@@ -698,6 +714,18 @@ class Meta:
     def get_root_project(self, obj) -> str:
         return str(obj.root_project.pk)
 
+    @extend_schema_field(OpenApiTypes.OBJECT)
+    def get_assigned_dev(self, obj):
+        return get_assigned_github_user(
+            project_id=obj.root_project.id, assigned_id=obj.assigned_dev
+        )
+
+    @extend_schema_field(OpenApiTypes.OBJECT)
+    def get_assigned_qa(self, obj):
+        return get_assigned_github_user(
+            project_id=obj.root_project.id, assigned_id=obj.assigned_qa
+        )
```

Comment:
@jerivas These should be `nullable` (and might as well extend GitHubCollaboratorSerializer?)

Reply:
Fixed in 9c2f3f8fdd1c41b649bb57845d02a532b760ee6a